dependencies.props

```diff
@@ -59,7 +59,7 @@
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>CoreFxVersion</ElementName>
-      <PackageId>Microsoft.Private.CoreFx.NETCoreApp</PackageId>
+      <PackageId>Microsoft.Private.CoreFx.UAP</PackageId>
```

Comment:
Nope, in release/uwp6.0 branch we turned off all of the packages that we won't be shipping in uwp release, which includes the netcoreapp package. That is what caused the corefx package version to stop getting updated. The file you are looking at in corefx is the one from master, if you check the one from release/uwp6.0 you'll see what I mean: https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/uwp6.0/Last_Build_Packages.txt